front-end/src/pages/FollowersPage.tsx

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import { Row, Col, Card, CardBody, CardTitle, Container } from 'reactstrap';
+import AuthorList from "../components/AuthorList"
+import AuthorListItem from '../components/AuthorListItem';
+import { Author } from '../types/Author';
+
+/**
+ * Render list of search results when searching for an author by display name
+ * @param props 
+ */
+export default function FollowersPage(props: any) {
```

Comment:
Try to type props if you can

https://github.com/CMPUT404W21T02/group-project/blob/main/front-end/src/pages/PostDetailPage.tsx might be a good reference

Reply:
done